readme.md

```diff
@@ -1,53 +1,54 @@
-
-The [NonPersistentObjectSpace/.AutoSetModifiedOnObjectChange](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.NonPersistentObjectSpace.AutoSetModifiedOnObjectChange) property is set to `true` to automatically mark non-persistent objects as modified when the `INotifyPropertyChanged.PropertyChanged`event is raised.
-
-We use a 'List<T>' as storage for Non-Persistent Object data.
+The [NonPersistentObjectSpace/.AutoSetModifiedOnObjectChange](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.NonPersistentObjectSpace.AutoSetModifiedOnObjectChange) property is set to `true` to automatically mark non-persistent objects as modified when XAF raises the `INotifyPropertyChanged.PropertyChanged` event.
 
+We use `List<T>` as storage for non-persistent object data.
 
 ### Common Components
 
 The following classes are used to provide a common functionality for all non-persistent objects used in the demo.
 
-#### NonPersistentObjectBase
-
-The abstract base class for all non-persistent objects used in the application. It provides a common implementation of the `INotifyPropertyChanged` and `IObjectSpaceLink` interfaces and some convenient protected methods.
-
-#### TransientNonPersistentObjectAdapter
-
-The adapter for transient (short-living) Non-Persistent business objects. Such objects exist only while their object space is alive. A new adapter instance is created for each non-persistent object space. It subscribes to object space events to manage a subset of object types in a common manner. It also maintains an identity map (`ObjectMap`) for `NonPersistentObjectSpace`.
-
-#### NonPersistentStorageBase
-
-Descendants of this class know how to create object instances and transfer data between objects and the storage. It knows nothing about the adapter. It also uses the identity map to avoid creating duplicated objects.
+- `NonPersistentObjectBase`  
+  The abstract base class for all non-persistent objects used in the application. It provides a common implementation of the `INotifyPropertyChanged` and `IObjectSpaceLink` interfaces and some convenient protected methods.
+  
+- `TransientNonPersistentObjectAdapter`  
+  The adapter for transient (short-living) non-persistent business objects. Such objects exist only while their object space is alive. A new adapter instance is created for each non-persistent object space. It subscribes to object space events to manage a subset of object types in a common manner. It also maintains an identity map (`ObjectMap`) for `NonPersistentObjectSpace`.
+  
+- `NonPersistentStorageBase`  
+  Descendants of this class know how to create object instances and transfer data between objects and the storage. It knows nothing about the adapter. It also uses the identity map to avoid creating duplicated objects.
```

Comment:
The first sentence starts with "descendants". So I think the second should start with "they". 

```suggestion
  Descendants of this class know how to create object instances and transfer data between objects and the storage. They know nothing about the adapter. They also use the identity map to avoid creating duplicated objects.
```